[["/usr/local/lib/python3.9/site-packages/exceptiongroup/__init__.py",[false,[["__all__",true,13,null],["catch",true,2,["catch","/usr/local/lib/python3.9/site-packages/exceptiongroup/_catch.py",12]],["BaseExceptionGroup",true,2,["BaseExceptionGroup","/usr/local/lib/python3.9/site-packages/exceptiongroup/_exceptions.py",5]],["ExceptionGroup",true,2,["ExceptionGroup","/usr/local/lib/python3.9/site-packages/exceptiongroup/_exceptions.py",5]],["format_exception",true,2,["format_exception","/usr/local/lib/python3.9/site-packages/exceptiongroup/_formatting.py",12]],["format_exception_only",true,2,["format_exception_only","/usr/local/lib/python3.9/site-packages/exceptiongroup/_formatting.py",12]],["print_exc",true,2,["print_exc","/usr/local/lib/python3.9/site-packages/exceptiongroup/_formatting.py",12]],["print_exception",true,2,["print_exception","/usr/local/lib/python3.9/site-packages/exceptiongroup/_formatting.py",12]]]]]]